File: LV1/1.3/nastyhacks/nastyhacks.py
n = int(input())

for _ in range(n):
    r, e, c = [int(i) for i in input().split(' ')]
    diff = e - r

    if diff == c:
        print("does not matter")
    elif diff > c:
        print("advertise")
    else:
        print("do not advertise")

File: LV1/1.3/nastyhacks/nastyhacks.rb
n = gets.to_i

n.times do
  r, e, c = gets.split.map(&:to_i)
  diff = e - r

  if diff == c
    puts 'does not matter'
  elsif diff > c
    puts 'advertise'
  else
    puts 'do not advertise'
  end
end
